src/cljs/nr/gameboard/player-stats.cljs

```diff
@@ -0,0 +1,127 @@
+(ns nr.gameboard.player-stats
+  (:require [clojure.string :as s :refer [capitalize join lower-case]]
+            [nr.appstate :refer [app-state]]
+            [nr.avatar :refer [avatar]]
+            [nr.gameboard.actions :refer [send-command]]
+            [nr.gameboard.utils :refer [inc-dec-overlay]]
+            [nr.gameboard.state :refer [game-state]]
+            [nr.translations :refer [tr tr-pronouns]]))
+
+(defn- inc-dec-overlay-for-side
+  [side]
+  (if (= (:side @game-state) side)
+    inc-dec-overlay
+    (fn [key content] content)))
+
+(defn- name-area
+  [user]
+  [:div.namearea [avatar user {:opts {:size 32}}]
+   [:div.namebox
+    [:div.username (:username user)]
+    (if-let [pronouns (get-in user [:options :pronouns])]
+      (let [pro-str (if (= "blank" pronouns) "" (tr-pronouns pronouns))]
+        [:div.pronouns (lower-case pro-str)]))]])
+
+(defmulti stats-area
+  (fn [player] {:side (get-in @player [:identity :side])
+                :icons (get-in @app-state [:options :player-stats-icons] true)}))
```

Comment:
This shouldn't be part of the multimethod. Inlining this check (or putting it as a cursor inside of a wrapping `let` block) is the better way, as it lets us re-use code instead of having mostly copied functions with only minor differences.

Reply:
You're right. There were more differences initially and I failed to refactor it again after I finished. Looks cleaner now.